src/corner_kick/src/types/visualizer.ts

```diff
@@ -1,9 +1,58 @@
-/**
- * This file describes a layer datatype
+/*
+ * This file describes the datatypes used for the visualizer
```

Comment:
Just a question, we only have 58 lines of types in the entire visualizer?

Reply:
No, this is just for the "visualizer" piece of the visualizer. We have more type files in the type directory